odoo-shippable/scripts/build-image.sh

```diff
@@ -244,7 +244,10 @@ EOF
 
 # Upgrade & configure vim
 apt install vim --only-upgrade
-wget -q -O /usr/share/vim/vim82/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/runtime/spell/es.utf-8.spl
+# Get vim version
+VIM_VERSION=$(dpkg -s vim | grep Version | sed -n 's/.*\([0-9]\+.[0-9]\+\)\..*/\1/p' | sed -r 's/\.//g')
+
+wget -q -O /usr/share/vim/vim${VIM_VERSION}/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/runtime/spell/es.utf-8.spl
```

Comment:
@ruiztulio 
FYI the last build is failing because of the following error:
 - [`/usr/share/vim/vim0~14/spell/es.utf-8.spl: No such file or directory`](https://hub.docker.com/repository/registry-1.docker.io/vauxoo/odoo-80-image-shippable-auto/builds/359fa31c-f2de-4620-ae97-3b2cae0a2f84)

Maybe something change here
Could you help me, please?